src/main/java/org/opensrp/queue/RabbitMQReceiver.java

```diff
@@ -0,0 +1,21 @@
+package org.opensrp.queue;
+
+import org.smartregister.pathevaluator.plan.PlanEvaluator;
+import org.springframework.amqp.core.AmqpTemplate;
+import org.springframework.amqp.core.Message;
+import org.springframework.amqp.core.MessageListener;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class RabbitMQReceiver implements MessageListener {
```

Comment:
is it a Spring Bean? Have we injected it via xml?

Reply:
Added annotation